src/main.ts

```diff
@@ -12,14 +12,10 @@ import 'vue-multiselect/dist/vue-multiselect.min.css'
 import 'scss/main.scss'
 
 import registerCustomComponents from '@/common-imports'
-import {BootstrapVue, BTooltip} from 'bootstrap-vue'
 
 import router from './router'
 import App from './App.vue'
 
-Vue.use(BootstrapVue)
```

Comment:
moving this back to vue component fixes the issue, so let's stick with that for now !

Reply:
Sounds good. We can get back to this (to how we register bootstrap vue) later :)